Fortinet FortiGate major bug fixes and improvements (#581)
rapid7/insightconnect-plugins: fortinet_fortigate/help.md

|Name|Type|Default|Required|Description|Enum|Example|
|----|----|-------|--------|-----------|----|-------|
|fqdn_filter|string|None|False|Optional FQDN to filter on|None|rapid7.com|
|name_filter|string|None|False|Optional name to filter on|None|MaliciousHost|
|subnet_filter|string|None|False|Optional subnet to filter on|None|198.51.100.100/32|

Example input:

```
{

>   "fqdn_filter": "rapid7.com",

Reviewer: ```suggestion
  "fqdn_filter": "example.com",
```

## Troubleshooting

> In order for this plugin to function the orchestrator's IP address must be added to the trusted hosts list for the API admin

Reviewer: The previous wording was better IMO

```suggestion
To accomplish this, log into the FortiGate firewall. Go to the System tab -> Administrator subtab and then select and edit the API admin.
```

> To accomplish this, log into the FortiGate firewall. Go to the System tab -> Administrator subtab and then select and edit the API admin account.

Reviewer 2: ```suggestion
```
Duplicate

> To do this log into the FortiGate firewall. Go to the System tab, Administrator subtab. Then select and edit the API admin.

Reviewer: ```suggestion
To accomplish this, log into the FortiGate firewall. Go to the System tab -> Administrator subtab and then select and edit the API admin account.
```

Add the orchestrator's IP address to the trusted hosts in CIDR form e.g. `198.51.100.100/32`

# Version History

> * 5.0.0 - Improve input handling to allow IPs, CIDRs, and subnet masks in actions | Fix output of get_address_objects to return searchable data | Update get_address_objects to allow for additional search parameters

Reviewer 2: ```suggestion
* 5.0.0 - Improve input handling to allow IPs, CIDRs, and subnet masks in actions | Fix output of Get Address Objects action to return usable data | Update Get Address Objects action to allow for additional search parameters
```

> * 5.0.0 - Improve input handling for all actions to allow IPs, CIDRs, and subnetmasks to all be handled correctly | fixed output of get_address_objects to return searchable data | Update get_address_objects to allow for additional search parameters

Reviewer: ```suggestion
* 5.0.0 - Improve input handling to allow IPs, CIDRs, and subnet masks in actions | Fix output of get_address_objects to return searchable data | Update get_address_objects to allow for additional search parameters
```
